Tests/Unit/Serializer/Jms/Handler/ResourceHandlerTest.php

```diff
@@ -84,6 +86,8 @@ public function testHandler()
                     'payload_alias' => 'alias',
                     'payload_type' => 'payload_type',
                     'path' => '/path/to/child',
+                    'resource_label' => 'child',
```

Comment:
Does it need the `resource_` prefix?` This seems to already represent the resource itself.